README.md

```diff
@@ -690,6 +691,21 @@ Use a descendant selector (preferably child '>' ) with the modified base classna
 ```
 
 
+<a name="multiple-classes-on-an-element"></a>
+### Multiple Classes on an Element
+
+In HTML, order classnames in a class attribute by order of object inheritance -- order in the class attribute does not matter functionally, but helps make the inheritance chain more apparent. 
+
+Separate multiple classes with TWO spaces for readability:
+
+```html
+<!-- Good -->
+<div class="o-inline-list  c-tutorial-nav  u-pull-left">
+
+<!-- Bad - don't use single space -->
+<div class="o-inline-list c-tutorial u-pull-left">
+```
+
```

Comment:
@Emma -- I could swear I added that in this morning. Might be in the wiki, I'll add it here.